region/brinstar/green.json

```diff
@@ -4177,6 +4291,98 @@
                     "Build up some run speed and then extend the Grapple Beam through the Blue Gate, while jumping, to open it from the wrong side.",
                     "This may take many attempts."
                   ]
+                },
+                {
+                  "name": "G-Mode Indirect Despawn Gate",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "indirect",
+                      "artificialMorph": false
+                    }}
+                  ]
+                }
+              ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "G-Mode Morph Indirect Despawn Gate - To the Top Right Door",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "indirect",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "canTrickyJump",
+                      {"and": [
+                        "Bombs",
+                        {"ammo": { "type": "PowerBomb", "count": 1 }}
+                      ]},
+                      {"ammo": { "type": "PowerBomb", "count": 3 }}
+                    ]},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "h_canArtificialMorphSpringBallBombJump"
```

Comment:
I needed 2 springball bomb jumps.  One to reach the the ledge with the bomb blocks, and another to get up to the door.